packages/react-native-aztec/android/build.gradle

```diff
@@ -116,34 +123,27 @@ dependencies {
     }
 }
 
-if (willPublishReactNativeAztecBinary) {
-    project.afterEvaluate {
-        publishing {
-            publications {
-                ReactNativeAztecPublication(MavenPublication) {
-                    artifact bundleReleaseAar
+project.afterEvaluate {
+    publishing {
+        publications {
+            ReactNativeAztecPublication(MavenPublication) {
+                artifact bundleReleaseAar
 
-                    groupId 'org.wordpress-mobile.gutenberg-mobile'
-                    artifactId 'react-native-aztec'
-                    version publishReactNativeAztecVersion
+                groupId 'org.wordpress-mobile.gutenberg-mobile'
+                artifactId 'react-native-aztec'
+                // version is set by 'publish-to-s3' plugin
 
-                    addDependenciesToPom(pom)
-                }
-            }
-            repositories {
-                maven {
-                    url "s3://a8c-libs.s3.amazonaws.com/android"
-                    credentials(AwsCredentials) {
-                        accessKey project.hasProperty('awsAccessKey') ? project.properties['awsAccessKey'] : System.getenv('AWS_ACCESS_KEY')
-                        secretKey project.hasProperty('awsSecretKey') ? project.properties['awsSecretKey'] : System.getenv('AWS_SECRET_KEY')
-                    }
-                }
+                addDependenciesToPom(pom)
```

Comment:
I'd like to remove this and the `artifact bundleReleaseAar` property and instead utilize `from.release` (which should add the necessary dependencies), but my initial test failed and I think it deserves its own PR, so I am leaving it as is.